shared-test-resources/TypeResolverTestSuite/ByteCodeHierarchyTest/src/Class2.java

```diff
@@ -0,0 +1,17 @@
+public class Class2 extends Class1 implements RootInterface2{
+
+    @Override
+    public void method1(){
+        System.out.println("I'm method 1 in Class2");
+    }
+
+    @Override
+    public void method2(){
+        System.out.println("I'm method2 in Class2");
```

Comment:
I think adding println to the test case will slow the test down. is this needed?